jest.preset.js

```diff
@@ -1,3 +1,8 @@
 const nxPreset = require('@nrwl/jest/preset');
 
-module.exports = { ...nxPreset };
+module.exports = {
+  ...nxPreset,
+  transformIgnorePatterns: [
+    'node_modules/(?!(react-native|my-project|redux-persist)/)',
```

Comment:
What is the purpose of this?

Reply:
when testing redux-persist throws an error and disturbs testing, this is the only solution i found